libqtopensesame/qtopensesame.py

```diff
@@ -940,6 +940,14 @@ def run_experiment(self, dummy=None, fullscreen=True, quick=False):
 		# Launch the runner!
 		if cfg.runner == u'multiprocess':
 			from libqtopensesame.runners import multiprocess_runner as runner
+
+			# Multiprocessing dus not work if opensesame is packaged as an app
+			# under OSX. For now just display a warning message and do nothing
+			# For the same reason, inOSX the default runner is set to inprocess 
+			# for now in misc.config
+			if sys.platform == "darwin" and getattr(sys, 'frozen', None):
```

Comment:
Before I merge, a small request: Could you use `self.experiment.notify()` to inform the user, instead of printing to the debug window? Also, if you use `print` statements (in other situations), please use the unicode-safe and Python 3 compatible syntax, i.e. `print(u'test')` and not `print 'test'`.